app.py

```
@@ -1,61 +1,405 @@
+TWITTER_API_KEY = os.getenv("TWITTER_API_KEY")
+TWITTER_API_SECRET = os.getenv("TWITTER_API_SECRET")
+TWITTER_ACCESS_TOKEN = os.getenv("TWITTER_ACCESS_TOKEN")
+TWITTER_ACCESS_TOKEN_SECRET = os.getenv("TWITTER_ACCESS_TOKEN_SECRET")
+DATABASE_PATH = os.getenv("DATABASE_PATH", "sc2sm.db")
+
+# Initialize OpenAI client
+openai_client = None
+if OPENAI_API_KEY and OPENAI_API_KEY != "your_openai_api_key_here":
+    try:
+        openai_client = OpenAI(api_key=OPENAI_API_KEY)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)